source/cluster-man/dovecot-cluster.7.rst.in

```diff
@@ -0,0 +1,72 @@
+.. _man-7_dovecot-cluster:
+
+===============
+dovecot-cluster
+===============
+
+
+NAME
+====
+
+dovecot-cluster - How dovecot cluster works
```

Comment:
Do we want this page at all? Shouldn't it belong to just doc.dovecot.org

Reply:
not 100% sure, maybe not? we have similar page for obox and pigeonhole.

Reply:
It's an extra location to remember to update when we change things. And we're going to forget to change them.. If there is a page, I think it should be very high level and generic which would never really be expected to change..